docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -53,27 +51,92 @@ When we added the OpenShift extension to the command line invocation above, the
     </dependency>
 ----
 
-By adding this dependency, we now have the ability to configure the OpenShift resource generation and application using the usual `application.properties` approach that Quarkus provides.
-The configuration items that are available can be found in: `io.quarkus.kubernetes.deployment.OpenShiftConfig` class.
-Furthermore, the items provided by `io.quarkus.deployment.ApplicationConfig` affect the OpenShift resources.
+== Log Into the OpenShift Cluster
+
+Before we build and deploy our application we need to log into an OpenShift cluster.
+You can log in via the https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI]:
+
+.Log In - OpenShift CLI Example
+[source,bash]
+----
+oc login -u myUsername <1>
+----
+<1> You'll be prompted for the required information such as server URL, password, etc.
+
+Alternatively, you may log in using the API token:
+
+.Log In - OpenShift CLI With API Token Example
+[source,bash]
+----
+oc login --token=myToken --server=myServerUrl
+----
+
+TIP: You can request the token via the _Copy Login Command_ link in the OpenShift web console.
+
+Finally, you don't need to use the OpenShift CLI at all.
+Instead, set the `quarkus.kubernetes-client.master-url` config property and authenticate with the `quarkus.kubernetes-client.token`, or `quarkus.kubernetes-client.username` and `quarkus.kubernetes-client.password` respectively.
+
+.Log In `application.properties` Example
+[source,properties]
+----
+quarkus.kubernetes-client.master-url=myServerUrl
```

Comment:
we shouldn't recommend users to put these in application properties should we ?

better do it using `mvn -Dquarkus.kubernets-client.token=... etc` ?

Reply:
Yes, that's a good point. Let's change the example...